industrial_taxonomy/pipeline/README.md

```diff
@@ -41,7 +41,7 @@ python industrial_taxonomy/pipeline/glass_description_ngrams/nlp_flow.py \
 
 ```bash
 python industrial_taxonomy/pipeline/glass_embed/flow.py \
- --environment=conda \
  --production \
+ --no-pylint \
  run
```

Comment:
I'd recommend running the whole flow with batch otherwise the upload of the org descriptions artifact in the start step (over home internet) to the S3 datastore is horribly slow.

```suggestion
 run \
 --with batch
```